diff --git a/mingw-w64-harfbuzz/PKGBUILD b/mingw-w64-harfbuzz/PKGBUILD
index 86a21defe..f6eff103a 100644
--- a/mingw-w64-harfbuzz/PKGBUILD
+++ b/mingw-w64-harfbuzz/PKGBUILD
@@ -21,9 +21,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-ninja"
              "${MINGW_PACKAGE_PREFIX}-ragel")
 depends=("${MINGW_PACKAGE_PREFIX}-freetype"
-         "${MINGW_PACKAGE_PREFIX}-gcc-libs"
-         "${MINGW_PACKAGE_PREFIX}-glib2"
-         "${MINGW_PACKAGE_PREFIX}-graphite2")
+         "${MINGW_PACKAGE_PREFIX}-gcc-libs")
 checkdepends=("${MINGW_PACKAGE_PREFIX}-python-fonttools" 
               "${MINGW_PACKAGE_PREFIX}-python-setuptools")
 options=('strip' 'staticlibs')
@@ -48,8 +46,8 @@ build() {
   mkdir -p "${_static_dir}" && pushd "${_static_dir}"
 
   # static build
-  CFLAGS="${CFLAGS} -DGRAPHITE2_STATIC" \
-  CXXFLAGS="${CXXFLAGS} -DGRAPHITE2_STATIC" \
+  CFLAGS="${CFLAGS}" \
+  CXXFLAGS="${CXXFLAGS}" \
   MSYS2_ARG_CONV_EXCL="--prefix=" \
   ${MINGW_PREFIX}/bin/meson \
     --prefix="${MINGW_PREFIX}" \
@@ -61,9 +59,10 @@ build() {
     -Dgobject=enabled \
     -Dicu=enabled \
     -Dgdi=enabled \
-    -Dgraphite=enabled \
+    -Dgraphite=disabled \
     -Dchafa=disabled \
     -Ddirectwrite=enabled \
+    -Dglib=disabled \
     ../${_realname}-${pkgver}
 
   ninja
@@ -84,9 +83,10 @@ build() {
     -Dgobject=enabled \
     -Dicu=enabled \
     -Dgdi=enabled \
-    -Dgraphite=enabled \
+    -Dgraphite=disabled \
     -Dchafa=disabled \
     -Ddirectwrite=enabled \
+    -Dglib=disabled \
     ../${_realname}-${pkgver}
 
   ninja
